libs/dh/market-participant/shell/src/lib/dh-market-participant-shell.routes.ts

```diff
@@ -30,9 +34,40 @@ import {
 import { DhMarketParticipantEditActorComponent } from '@energinet-datahub/dh/market-participant/edit-actor';
 import { DhMarketParticipantGridAreaComponent } from '@energinet-datahub/dh/market-participant/feature-gridarea';
 import { PermissionGuard } from '@energinet-datahub/dh/shared/feature-authorization';
-import { Routes } from '@angular/router';
+import { DhFeatureFlagsService } from '@energinet-datahub/dh/shared/feature-flags';
 
 export const dhMarketParticipantShellRoutes: Routes = [
+  {
+    path: '',
+    pathMatch: 'full',
+    redirectTo: dhMarketParticipantActorsPath,
+  },
+  {
+    path: dhMarketParticipantActorsPath,
+    canMatch: [
+      () => {
```

Comment:
Question: Is it okay that this function has two diffent return types

Reply:
Yes. returning `true` means that the navigation can proceed. Returning a `UrlTree` means that Angular will cancel the current navigation and make a redirect to another page.